app/views/govuk_publishing_components/components/docs/glance_metric.yml

```diff
@@ -0,0 +1,23 @@
+name: "Glance metric"
+description: "The glance metric component is an at-a-glance view of data for a content item. The minimum requirements for it to display are a name and figure. "
+part_of_admin_layout: true
+body: |
+
+accessibility_criteria: |
+  - Where an abbreviated label for a metric is used it MUST be accompanied with the explicit label
```

Comment:
Can you add in the accessibility acceptance criteria that the element must be under a suitable `<h2>`, as they are using `<h3>` elements, and if the heading structure of the page breaks it becomes an accessibility issue. 

It may be worth tweaking the current criteria you've written to add what the big number component mentions too: https://components.publishing.service.gov.uk/component-guide/big_number#accessibility-acceptance-criteria

Reply:
Good call, I've made a tweak so the `heading_level` can be customised if necessary